deploy/example_catalog/docs/elk771-readme.md

```diff
@@ -92,4 +92,12 @@ More about the ELK cluster can be read [here.](https://www.elastic.co/what-is/el
         }
        }
 
-   Similarly we can perform update, delete on the product indice by sending rest request from the console tool in Kibana.
\ No newline at end of file
+   Similarly we can perform update, delete on the product indice by sending rest request from the console tool in Kibana.
+
+#### Docker Image location
+
+* docker.io/bluedata/elk771
+
+#### Docker Pull command
+
+* docker pull docker.io/bluedata/elk771:1.1
```

Comment:
same question here for 1.1 tag

Reply:
@kmathur2 The change is only in the documentation with the latest tag of the elk image which is 1.1.